.claude/TM_COMMANDS_GUIDE.md

```
@@ -0,0 +1,147 @@
+/project:tm/set-status/to-done <id>
+```
+
+### Task Breakdown
+```
+/project:tm/show <id>
+/project:tm/expand <id>
+/project:tm/list/with-subtasks
+```
+
+### Sprint Planning
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)